README.rst

```diff
@@ -81,36 +129,147 @@ Creating a Cluster
 Running a Relay
 ---------------
 
+Now, with the replication set all set up, you can run a relay process.
+
 ::
 
     pgshovel-relay configuration.yml example
 
+The configuration file defines the relay object that should be run, as well its configuration.
+
+
+Replication Worker
+------------------
+
+A replication worker is used to replicate a stream of batch operations into a new database replication target. It is started with the ``pgshovel-replicate`` command, passing the path to the replication config file as well as the replicaion set name.
+
+::
+
+    python-replicate example/configurations/replication.yml example
+
+The config file looks very similar to the relay config file, though it contains three sections: ``loader``, ``stream``, and ``target``, corresponding to the three components of replication as described in the "Concepts" section above.
+
+The replication feature is ongoing, so further documentation of the API is unavailable until the API becomes stable.
+
+Results
+-------
+
+That's it!  The tables defined in the ``example`` replication set now have their mutations replicated out of PostgreSQL by the relay.  Additionally, the replication worker bootstrapped a new target and is now applying any new mutations to the target data store.
+
+Further Administration
+======================
+
+``pgshovel`` also supports updates to its configuration via a variety of tasks.
+
 Updating a Replication Set
 --------------------------
 
+If for any reason you need to update a replication set, you may do so using the ``update`` command, replacing the existing configuration with the one contained in the new config file.
+
 ::
 
-    pgshovel-admin set upgrade /path/to/configuration.pgshovel
+    pgshovel-admin set update /path/to/configuration.pgshovel
 
 Dropping a Replication Set
 --------------------------
 
+Replication sets may be dropped via the ``drop`` command.
+
 ::
 
     pgshovel-admin set drop example
 
-Configuration
-=============
+Further Configuration and Development
+======================================
 
-.. todo:: Rewrite with config file
+In order to run a relay or replication worker, you need to specify a configuration file their respective command line tool. This section documents the format of those configuration files, as well as information on writing your own components for both systems.
 
-Operations
-==========
+Batch Operations
```

Comment:
@fuziontech @ivanov please let me know if this section makes sense to you. @fuziontech this is important in regards to building a replication target and I want to make sure these words make sense. I had a hard time writing this section.